pkgs/development/tools/build-managers/buildbot/plugins.nix

```diff
@@ -26,14 +26,14 @@ in {
   www = pythonPackages.buildPythonPackage rec {
     name = "${pname}-${version}";
     pname = "buildbot_www";
-    version = "0.9.0.post1";
+    version = "0.9.3";
 
     # NOTE: wheel is used due to buildbot circular dependency
     format = "wheel";
 
     src = fetchurl {
-      url = "https://pypi.python.org/packages/02/d0/fc56ee27a09498638a47dcc5637ee5412ab7a67bfb4b3ff47e041f3d7b66/${name}-py2-none-any.whl";
```

Comment:
That's because the package is named `buildbot-www` and not `buildbot_www`.
This is a bit frustrating, Nixpkgs guidelines say the name should be `buildbot_www` but in pythonPackages I push for the original name.